Firestore/tests/System/FirestoreSessionHandlerTest.php

```diff
@@ -59,25 +59,69 @@ public function testSessionHandler()
         $this->assertTrue($hasDocument);
     }
 
-    public function testSessionHandlerGarbageCollection()
+    public function testGarbageCollection()
     {
         $client = self::$client;
 
+        // Set session max lifetime to 0 to ensure deletion
+        ini_set('session.gc_maxlifetime', 0);
```

Comment:
We should also add `ini_set('session.gc_probability', 0);` to make sure `session_start` doesn't run gc and interfere with `session_gc`.